src/3d/qgs3dmapsettings.h

```diff
@@ -426,6 +412,16 @@ class _3D_EXPORT Qgs3DMapSettings : public QObject, public QgsTemporalRangeObjec
      */
     double outputDpi() const { return mDpi; }
 
+    /**
+     * Returns the current configuration of the skybox
+     */
+    QgsSkyboxSettings skyboxSettings() const SIP_SKIP { return mSkyboxSettings; }
```

Comment:
It's because when I changed the skybox type to being an enum I both didn't know how to correctly export it and the enum was inside QgsSkyboxEntity class that inherits from Qt3DCore::QEntity which wasn't available for python bindings.